yellowbrick/cluster/elbow.py

```diff
@@ -271,8 +282,10 @@ def draw(self):
         Draw the elbow curve for the specified scores and values of K.
         """
         # Plot the silhouette score against k
-        self.ax.plot(self.k_values_, self.k_scores_, marker="D", label="score")
-
+        self.ax.plot(self.k_values_, self.k_scores_, marker="D")
+        if self.knee:
+            self.ax.axvline(self.knee_value,c='black',linestyle='--')
```

Comment:
For the color, would you mind using [`yellowbrick.style. palettes.LINE_COLOR`](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/style/palettes.py#L45)?

Reply:
I'll use that.